onnxruntime/__init__.py

```diff
@@ -76,3 +76,76 @@
     __version__ = version
 
 onnxruntime_validation.check_distro_info()
+
+# Load nvidia libraries from site-packages/nvidia if the package is onnxruntime-gpu
+if (
+    __package__ == "onnxruntime-gpu"
+    # Just in case we rename the package name in the future
+    or __package__ == "onnxruntime-cuda"
+    or __package__ == "onnxruntime_gpu"
+    or __package__ == "onnxruntime_cuda"
+):
+    import ctypes
+    import os
+    import platform
+    import re
+    import site
+
+    # Get the site-packages path where nvidia packages are installed
+    site_packages_path = site.getsitepackages()[0]
```

Comment:
This one is not the right one in Conda:
```
>>> site.getsitepackages()
['C:\\Users\\tlwu\\.conda\\envs\\py310', 'C:\\Users\\tlwu\\.conda\\envs\\py310\\lib\\site-packages']
```

To fix it, we can either check each path in the list, and get first one that ends with site-packages.

Or compose it like
```
site_packages_path = os.path.join(sys.prefix, "lib", "python{0}.{1}".format(*sys.version_info[:2]), "site-packages", "nvidia")
```

Reply:
Updated the usage to `site.getsitepackages()[-1]` and lets give it a try